### PR TITLE
add header attributes for all files

### PR DESCRIPTION
**Pull request by eshaben**
This PR adds attributes to each of the header sections so that in the language repos headers can be translated and links to each header/section will still work. 

Created using the script thats in [moonbeam-mkdocs PR #18](https://github.com/PureStake/moonbeam-mkdocs/pull/18)

### CODE REVIEW

File: resources/glossary.md [hunk 1]
```diff
@@ -7,50 +7,50 @@ description: We've compiled a glossary of terms related to Polkadot that'll make
...
 
 One of the key network participants needed to support parachains within the Polkadot Network.  In Moonbeam, collators are the nodes that are responsible for block production and for submitting produced blocks up to the Polkadot relay chain for finalization.
 
-### Nominators
+### Nominators {: #nominators } 
 
 Token holders who select to "back" a validator. They can receive part of the validator's reward, but are subject to slashing of their staked tokens in case the validator misbehaves. A nominator can back up to 16 validators, and their bond is fully distributed between the backed validators that were selected for the validator set. If you want to nominate PureStake for both Polkadot and/or Kusama, please follow [this guide](https://www.purestake.com/technology/polkadot-validator/).
 
-### Nomitated Proof of Stake
+### Nomitated Proof of Stake {: #nomitated-proof-of-stake }
```
Comment (themacexpert) [inline resources/glossary.md:18]: Typo here (not from your changes but would be good to fix at same time)

File: learn/xchain-plans.md [hunk 2]
```diff
@@ -9,21 +9,21 @@ One of the key planned features for Moonbeam is to provide an easy way for devel
 
 Polkadot defines a low-level integration protocol, called cross-chain message passing (XCMP), that can be used to facilitate communication between parachains within the Polkadot network and share trusted logic between chains on the Polkadot network, called shared protected runtime execution enclaves (SPREE). Parity is in the process of implementing XCMP and is in the design phase of SPREE as of writing this document (July 2020). Post-Polkadot mainnet launch, XCMP and SPREE support will be released as upgrades to the Polkadot relay chain. We plan to implement and support integration scenarios based on these protocols when they are available.
 
-## An Analogy: The "Bash of Polkadot"
+## An Analogy: The "Bash of Polkadot" {: #an-analogy:-the-"bash-of-polkadot" }
```
Comment (themacexpert) [inline learn/xchain-plans.md:12]: I would double check the formatting here to make sure it's correct

**Review by themacexpert — APPROVED** [review]
LGTM, 👍   just two minor comments

File: learn/xchain-plans.md (hunk 2 above)
Comment (eshaben) [reply learn/xchain-plans.md]: good catch! 🐟 

I was able to fix a lot of mistakes and I think I got them all!

**Pull request merged**